src/spec-node/featuresCLI/package.ts

```diff
@@ -0,0 +1,90 @@
+import path from 'path';
+import { Argv } from 'yargs';
+import { CLIHost, getCLIHost } from '../../spec-common/cliHost';
+import { loadNativeModule } from '../../spec-common/commonUtils';
+import { Log, LogLevel, mapLogLevel } from '../../spec-utils/log';
+import { isLocalFolder, mkdirpLocal } from '../../spec-utils/pfs';
+import { createLog } from '../devContainers';
+import { UnpackArgv } from '../devContainersSpecCLI';
+import { getPackageConfig } from '../utils';
+import { doFeaturesPackageCommand } from './packageCommandImpl';
+
+export function featuresPackageOptions(y: Argv) {
+    return y
+        .options({
+            // 'features': { type: 'array', alias: 'f', describe: 'Feature(s) to test as space-separated parameters. Omit to auto-detect all features in collection directory.  Cannot be combined with \'-s\'.', },
+            'features-folder': { type: 'string', alias: 'c', default: '.', description: 'Path to folder containing features source code' },
+            'output-dir': { type: 'string', alias: 'o', default: '/tmp/build', description: 'Path to output' },
```

Comment:
Should the default also be the current working directory?

Reply:
maybe it generates an `output` directory in the output dir? I didn't want it to just spit out 20 tgz files in the root.